plugin/helpers_bots.go

```diff
@@ -10,6 +10,12 @@ import (
 )
 
 func (p *HelpersImpl) EnsureBot(bot *model.Bot) (retBotId string, retErr error) {
+	const minimumSupportedVersion = "5.10"
+
+	if p.API.GetServerVersion() < minimumSupportedVersion {
```

Comment:
@hanzei I don't see any point in making these code as a method.

Reply:
You will need to apply this check to also other helpers. Hence, you want to keep your code DRY. This is why it should be a method IMO.

Reply:
Done.